lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -122,23 +84,37 @@ def endpoint
 
     def fulfill(order_id, shipping_address, line_items, options = {})
       requires!(options, :order_date, :shipping_method)
-      commit :post, :outbound, :create, build_fulfillment_request(order_id, shipping_address, line_items, options)
+      with_error_handling do
+        data = commit :post, 'CreateFulfillmentOrder', build_fulfillment_request(order_id, shipping_address, line_items, options)
+        parse_fulfillment_response(parse_document(data), 'Successfully submitted the order')
```

Comment:
Moves all the parsing back to the top level methods, making it more clear as to what's happening.